endpointmanager/pkg/chplendpointquerier/bundleparser.go

```diff
@@ -0,0 +1,40 @@
+package chplendpointquerier
```

Comment:
Okay great, thanks!

Reply:
I realize why I didn't originally use the existing parser now that I'm trying to use it.
a. The CHPL endpoint data is stored in a different format in the json files than the regular endpoints
b. The other CHPL files use different struct types for handling endpoints than what is used by the existing parser e.g. LanternEntry vs. EndpointEntry
c. I need to do type changing and json unmarshalling to get the bundle response into a []map[string]interface{} to be processed by the parser

I can use the existing parser so long as I do type conversions before and after calling it. Would you prefer that I do that rather than use the separate parser I made that uses the same types as the CHPL format?